csrfguard/src/main/java/org/owasp/csrfguard/config/PropertiesConfigurationProvider.java

```diff
@@ -563,7 +564,11 @@ private void javascriptInitParamsIfNeeded() {
 				} else if (servletConfig.getServletContext().getRealPath(this.javascriptSourceFile) != null) {
 					this.javascriptTemplateCode = CsrfGuardUtils.readFileContent(servletConfig.getServletContext().getRealPath(this.javascriptSourceFile));
 				} else {
-					throw new IllegalStateException("getRealPath failed for file " + this.javascriptSourceFile);
+					try( InputStream inputStream = servletConfig.getServletContext().getResourceAsStream("/" + this.javascriptSourceFile)) {
```

Comment:
In python might be considered "pythonic" to catch NPEs this way, but in Java it's not considered a good practice. Please do a proper null check instead and narrow the exception scope.